### PR TITLE
Exercise 4

### PR DESCRIPTION
**Pull request by josephcwalker**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,22 @@ Press CTRL+C to quit
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+## Deploying the App
+
+Deployment is managed with Ansible.
+
+Create an [inventory file](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html)
+with a `webservers` group containing all the hosts to deploy the app to.
```
Comment (josephcwalker) [inline README.md:79]: it felt weird to commit the inventory file as it contained the IP of the host to deploy to

what's standard here?

Comment (JackMeadDev) [reply README.md:79]: For this exercise, just committing it is standard (if nothing else, it makes my life a bit easier remembering which host to check! 😛 ). 
For a real project, you're right, I wouldn't want the server IPs in a public repo certainly (I don't think I'd be _very_ fussed in a private one, but might depend on sensitivity). The answer to what's standard likely depends on what happens next though, e.g.:
* If your servers are being created by Terraform, then potentially just generate the hosts file programmatically using Terraform output
* If you have a pipeline that runs your Ansible against fixed addresses, maybe you inject them as pipeline secrets somehow
* If you're running them manually on a controller, maybe you just leave the file on the controller (and record it in your docs somewhere)
etc.

File: playbook.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        creates: ~/.local/bin/poetry
+
+    - name: Create TODO App Directory
+      become: true
+      file:
+        path: /opt/todoapp
+        state: directory
+        owner: ec2-user
+
+    - name: Clone TODO App
```
Comment (JackMeadDev) [inline playbook.yaml:44]: This stuck out so hard to me like a leftover `#TODO` comment - brain couldn't parse it at first!

**Review by JackMeadDev — APPROVED** [review]
This looks great, clean effective playbook and I can see the app's running happily on your managed node! 😄 No suggestions from me, but let me know if my answer on good practice for the inventories misses the mark - otherwise very happy for you to merge when you're ready!

**Pull request merged**